tensorflow/compiler/xla/service/gpu/alpa_nccl_wrapper.h

```diff
@@ -70,29 +69,35 @@ ncclDataType_t ToNcclDataType(PrimitiveType element_type);
 
 int SizeOfType(ncclDataType_t element_type);
 
-StatusOr< std::shared_ptr< std::vector<ncclComm_t> > > NcclInitCommunicator(std::vector<int> devices_vec);
+class NcclCommStorage {
```

Comment:
Nit: `Storage` is a general name that does not indicate what this part is doing, can you rename it to something more concrete? Maybe `NcclCommAndStreamVector`?

Reply:
OK

Reply:
Well, I might add other information(e.g. cuda event) in this storage later, so I use the word `storage` here.